datasets/safecast.yaml

```diff
@@ -0,0 +1,28 @@
+Name: Safecast
+Description: An ongoing collection of radiation and air quality measurements taken by devices involved in the Safecast project.
+Documentation: https://github.com/Safecast/safecastapi/wiki/Data-Sets
+Contact: https://groups.google.com/forum/#!forum/safecast-devices
+ManagedBy: "[Safecast](https://safecast.org/)"
+UpdateFrequency: Continuous
+Tags:
+  - air
```

Comment:
Hi @matschaffer please note that we don't have an `air` tag, but we do have `air quality`. Would that work?